src/utils/ssmCommand.ts

```diff
@@ -13,7 +13,7 @@ export function ssmCommand(command: string, instanceId: string, outputExpected:
             .then((result: SSM.Types.GetCommandInvocationResult) => {
                 if (result.Status === 'Success' && outputExpected) return (result.StandardOutputUrl);
                 else if (result.Status === 'Success' && !outputExpected) resolve("success");
-                else reject(`SSM command result was: ${result.Status} / ${result.StandardErrorContent}`)
+                else reject(`SSM command: ${command} result was: ${result.Status} / ${result.StandardErrorContent}`)
```

Comment:
As the command can be quite a long (possibly-multi-line) string, it's probably a good idea to put it last, so that the shorter pieces of data can be read more easily before the command splashes everywhere:

```suggestion
                else reject(`SSM command executed on ${instanceId} not successfully completed. status='${result.Status}' errorContent='${result.StandardErrorContent}' command='${command}'`)
```